### PR TITLE
Add stringify to json objects

### PR DESCRIPTION
**Pull request by yaron-idan**
Fixes issue with decrypting json files.
Not sure about keeping this in a lambda as it's a multi liner now, feedback appreciated.

### CODE REVIEW

File: init-container/index.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const decryptFile = async (httpClient, filePath, folder) => {
     var encryptedContent = await readFileAsync(folder + '/' + filePath, "utf8");
     try {
       const response = await httpClient.post('/api/v1/decrypt', {data: encryptedContent});
+      console.log(response.data);
```
Comment (omerlh) [inline init-container/index.js]: A mistake?

File: init-container/index.js [hunk 2]
```diff
@@ -106,7 +106,10 @@ async function innerRun() {
         await writeFile(outputFile, serializeToCfgFormatStrict(secrets));
         break;
       case "files":
-        await Promise.all(Object.keys(secrets).map(secretName => writeFile(path.join(program.decryptedPath, secretName), secrets[secretName])))
+        await Promise.all(Object.keys(secrets).map(secretName => writeFile(path.join(program.decryptedPath, secretName),
```
Comment (omerlh) [inline init-container/index.js]: There is a similar code on this [line](https://github.com/Soluto/kamus/blob/b69715467deab9af4f2cf9b505dbc8072fac4ba8/init-container/index.js#L59), maybe worth reusing it?

File: init-container/index.js (hunk 1 above)
Comment (yaron-idan) [reply init-container/index.js]: Already pushed a commit

File: init-container/index.js (hunk 2 above)
Comment (yaron-idan) [reply init-container/index.js]: It actually looks like it's doing the opposite of my addition.
I can port my changes to a separate method but I don't think the code can be reused in the current form.

Comment (omerlh) [reply init-container/index.js]: The constructor check is a bit magical to me, maybe worth using `typeof`?

Comment (yaron-idan) [reply init-container/index.js]: Good point, I've made the change. Do you also think it's worth moving to a function?

Comment (omerlh) [reply init-container/index.js]: Nope, looks better now :)

**Pull request merged**